xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -1337,13 +1337,28 @@ You can use asynchronous read operations to avoid these dependencies and their d
       <AssemblyInfo>
         <AssemblyName>System</AssemblyName>
       </AssemblyInfo>
+      <Attributes>
```

Comment:
These lines are supposed to be autogenerated. Where did you get them from?

Reply:
I copied them from `StandardOutputEncoding`: https://github.com/dotnet/dotnet-api-docs/pull/3227/files#diff-283b6652f917fe3bda6d93ccb3e1d63bR1389

Reply:
This is wrong. This API only exists in .NET Core and .NET Standard only. I'll remove that.